src/main/java/com/uid2/admin/Main.java

```diff
@@ -228,6 +228,14 @@ public void run() {
             RotatingKeysetProvider keysetProvider = new RotatingKeysetProvider(cloudStorage, keysetGlobalScope);
             KeysetStoreWriter keysetStoreWriter = new KeysetStoreWriter(keysetProvider, fileManager, jsonWriter, versionGenerator, clock, keysetGlobalScope, enableKeysets);
 
+            if(enableKeysets) {
```

Comment:
Can you add a large comment here about why it's important that this code had to be moved? What bad thing happens in the original place etc. I'd hope for a paragraph or so.